debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SourceInfo.java

```diff
@@ -31,17 +28,15 @@
     public static final String SNAPSHOT_KEY = "snapshot";
     public static final String TIMESTAMP_KEY = "ts_micro";
 
-    public static final Schema SOURCE_SCHEMA = SchemaBuilder.builder().record(SOURCE).fields()
-            .requiredString(DEBEZIUM_VERSION_KEY)
```

Comment:
ah, I suppose given the connector is incubating it doesn't make sense to dbz version key right now, we'd probably want to add `version` and `connector` back when it graduates for consistency

Reply:
I copied my code changes from cassandra-cdc repo to debezium repo thus didn't realized that I removed these lines, added them back :)